src/docs/product/accounts/sso/index.mdx

```diff
@@ -27,7 +27,7 @@ Every member who creates a new account via SSO will be given global organization
 
 Our SSO implementation prioritizes security. We aggressively monitor linked accounts and will disable them within any reasonable sign that the account’s access may have been revoked. Generally this will be transparent to you, but if the provider is functioning in an unexpected way you may experience more frequent re-authorization requests.
 
-Sessions last for [Django's default session length](https://docs.djangoproject.com/en/1.11/topics/http/sessions/#using-cookie-based-sessions), which is 2 weeks. We do not support customizing the session length.
+Sessions last for [Django's default session length](https://docs.djangoproject.com/en/1.11/topics/http/sessions/#using-cookie-based-sessions), which is 2 weeks. For individual organizations with Single Sign On, we expire organization access after one day (20 hours).
```

Comment:
"single sign-on" shouldn't be capitalized. I'll do a sweep for this in the SSO docs separate from this PR.

```suggestion
Sessions last for [Django's default session length](https://docs.djangoproject.com/en/1.11/topics/http/sessions/#using-cookie-based-sessions), which is two weeks. For individual organizations with single sign-on, we expire organization access after one day (20 hours).
```